docs/en/index.rst

```diff
@@ -16,6 +16,17 @@ You can install the Doctrine RST Parser with composer:
 Basic Usage
 -----------
 
+Then you can run the following script on the console:
+
+.. code-block:: console
+
+    $ vendor/bin/doctrine-rstparser render "my/path/to/documentation" "my/path/to/output"
```

Comment:
```suggestion
    $ vendor/bin/doctrine-rstparser render path/to/documentation path/to/output
```

Reply:
I wonder if "documentation" is the right word here… I mean the output is also documentation, right? Maybe "source"?